src/crimson/mgr/client.cc

```diff
@@ -168,6 +170,34 @@ void Client::report()
   });
 }
 
+void Client::_send_report()
+{
+  // TODO: implement daemon_health_metrics support
+  // https://tracker.ceph.com/issues/63766
+  gate.dispatch_in_background(__func__, *this, [this] {
+    if (!conn) {
+      logger().warn("cannot send report; no conn available");
+      return seastar::now();
+    }
+    auto report = make_message<MMgrReport>();
+    // Adding empty information since we don't support perfcounters yet
+    report->undeclare_types.emplace_back();
+    ENCODE_START(1, 1, report->packed);
+    report->declare_types.emplace_back();
+    ENCODE_FINISH(report->packed);
```

Comment:
@Matan-B addressed your review comment to avoid changing classical code. Does this look okay to you?

Reply:
Yes, we will have to verify that the mgr is ok with empty stats, haven't seen any asserts so it should be fine.